generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java

```diff
@@ -172,6 +173,102 @@ public OpaqueExpressionGenerator(IRandom generator, GenerationParams generationP
         depth, fuzzer)));
   }
 
+  /**
```

Comment:
I really appreciate the thoroughness of this method comment - nice work!

Reply:
Thanks - I was a little puzzled by how to go about this in the beginning, so I ended up writing the comments before writing the function, which helped quite a bit.